cmd/commands/client/command_client.go

```diff
@@ -51,13 +52,19 @@ func NewCommandWith(
 
 	statsKeeper := bytescount.NewSessionStatsKeeper(time.Now)
 
+	ipResolver := ip.NewResolver(options.IpifyUrl)
+
+	locationDetector := location.NewDetector(filepath.Join(options.DirectoryConfig, options.LocationDatabase))
```

Comment:
We need to start distributing DB file together with `mysterium_client`:
- `bin/client_package_debian`
- `bin/client_docker/alpine/Dockerfile`